R/addins.R

```diff
@@ -1,8 +1,115 @@
-read_addin <- function(...) {
-  path <- fs::path_package("rhino", "rstudio", "addins", ...)
-  readChar(path, file.info(path)$size)
+run_background <- function(file) {
+  path <- fs::path_package("rhino", "rstudio", "addins", file)
+  rstudioapi::jobRunScript(path, workingDir = rstudioapi::getActiveProject())
 }
 
 addin_module <- function() {
-  rstudioapi::insertText(read_addin("module.R"))
+  module_path <- fs::path_package("rhino", "rstudio", "addins", "module.R")
+  name <- rstudioapi::showPrompt(
+    title = "Module name",
+    message =  "Please set module name"
+  )
+  file_path <- glue::glue("app/view/{name}.R")
```

Comment:
For constructing paths, use `fs::path`. It will take care about system specifics, e.g. `/` vs `\`.

Reply:
Updated! Thanks!